### DISCUSSION TITLE
Any way to still use Gamepad/Controller support instead of VR controllers?

### DISCUSSION THREAD

**Discussion by CROFTEA** [Okabintaro/SubmersedVR#134] [Q&A] [answered]
As the title says: anyway to still use Gamepad/Controller support instead of VR controllers?
Ive tried launching in flatscreen mode first and checking gamepad option is ticked (then started in VR again)
I know the gamepad is working as it works with other games and the XBOX button on the controller works when the game is launched...but nothing else. 

Would love this as a feature as I much prefer playing with a controller. 
Thanks!

**Comment by Okabintaro** [comment, marked as answer]
The goal of the mod is to add motion controls.
All modifications made require those.

If you just wanna play with a controller I recommend the VR enhancements mod(see Readme).

**Reply by CROFTEA** [reply]
Thank you for taking the time to reply :)
I did see VR enhancements, but I assumed that was just for the first subnautica?
